common/app/model/dotcomrendering/pageElements/PageElement.scala

```diff
@@ -1219,7 +1219,7 @@ object PageElement {
             val imageOverride = overrideImage.map(_.images).flatMap(Video700.bestSrcFor)
             val altText = overrideImage.flatMap(_.images.allImages.headOption.flatMap(_.altText))
             mediaAtom match {
-              case youtube if mediaAtom.assets.headOption.exists(_.platform == MediaAssetPlatform.Youtube) => {
+              case youtube if mediaAtom.activeAssets.headOption.exists(_.platform == MediaAssetPlatform.Youtube) => {
```

Comment:
Given the `headOption` here I assume there is only ever 1 active asset

Reply:
I've confirmed in Media Atom Maker that only one asset can be active at a time. This is the case regardless of whether the active asset is a youtube asset or a self hosted video.